pipeline/genomeBrowserTrack/brcaExchange.as

```diff
@@ -0,0 +1,142 @@
+table brcaExchanges
+" bed9 + many additional fields from brcaExchange.org "
+    (
+    string chrom;      "Chromosome (or contig, scaffold, etc.)"
```

Comment:
im referring to the fields in general, not the chrom field specifically. For example, new fields may be added or field names may change -- in a recent change "Minor_allele_frequency_ESP_percent" was changed to "Minor_allele_frequency_percent_ESP"